cleanlab/datalab/issue_manager/noniid.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         metric: Optional[str] = None,
         k: int = 10,
         num_permutations: int = 25,
+        seed: Optional[int] = None,
```

Comment:
why choose default None? Don't we want this to be deterministic by default?
We should still allow user to pass None if they dont want to set seed, but my suggestion is to set default = 0 say

Reply:
Changed behavior to be deterministic by default.